<launch>

    <!-- GMapping -->
    <node pkg="gmapping" type="slam_gmapping" name="gmapping" output="screen">
        <remap from="scan" to="scan_multi" />
        <param name="map_frame" value="/map" />
        <param name="odom_frame" value="/odom" />
        <param name="base_frame" value="/base_link" />
        <param name="map_update_interval" value="2.0" />
        <param name="linearUpdate" value="0.5" />
        <param name="xmin" value="-15.0" />
        <param name="ymin" value="-15.0" />
        <param name="xmax" value="15.0" />
        <param name="ymax" value="15.0" />
        <param name="delta" value="0.05" />
    </node>

    <!-- Explore lite -->
    <node pkg="explore_lite" type="explore" name="explore" output="screen">
        <param name="robot_base_frame" value="base_link"/>
        <param name="costmap_topic" value="map"/>
        <param name="costmap_updates_topic" value="map_updates"/>
        <param name="visualize" value="true"/>
        <param name="planner_frequency" value="1.0"/>
        <param name="progress_timeout" value="30.0"/>
        <param name="potential_scale" value="2.0"/>
        <param name="orientation_scale" value="1.0"/>
        <param name="gain_scale" value="1.0"/>
        <param name="transform_tolerance" value="0.3"/>
        <param name="min_frontier_size" value="0.2"/>
    </node>

</launch>